app/src/main/java/edu/uc/willi6jd/mycaloriecounter/ui/main/foodView/FoodActivity.kt

```diff
@@ -15,13 +15,13 @@ import edu.uc.willi6jd.mycaloriecounter.R
  * This will control the view when they get to the food page
  */
 class FoodActivity : AppCompatActivity() {
-    var foodList = ArrayList<Food>()
+    //var foodList = ArrayList<Food>()
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_food)
         val listView = findViewById<ListView>(R.id.food_listview)
         listView.adapter =
-            foodCustomerAdapter(
```

Comment:
it is a known naming convention to start classes with uppercase letters.

Reply:
Yes, class names should be UpperCamelCase.